# Service that can be used to set the current EE pose and control the robot using
# MoveIt.
geometry_msgs/Pose pose
---
bool success
string message
